portal/templates/admin_base.html

```diff
@@ -0,0 +1,8 @@
+{%- extends "layout.html" -%}
+{% block additional_scripts %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jasny-bootstrap/3.1.3/js/jasny-bootstrap.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.11.1/extensions/export/bootstrap-table-export.min.js"></script>
+<script src="//rawgit.com/hhurz/tableExport.jquery.plugin/master/tableExport.js" async></script>
```

Comment:
We'll have to use [something else](https://alternativeto.net/software/rawgit/) (as suggested by the review). Could you pin this to a [specific release](https://github.com/hhurz/tableExport.jquery.plugin/releases)? master will change over time